scripts/reset.mjs

```diff
@@ -0,0 +1,29 @@
+// Resets the repository by deleting all untracked files except for few exceptions.
```

Comment:
maybe we should add `process.env.FORCE_COLOR='1'` to pipe [subprocess TTY colors correctly](https://google.github.io/zx/known-issues#colors-in-subprocess)